doc/config.example.yaml

```diff
@@ -85,7 +85,7 @@ validator:
   # DO NOT USE THOSE VALUES ANYWHERE
   # Private seed:    SCT4KKJNYLTQO4TVDPVJQZEONTVVW66YLRWAINWI3FZDY7U4JS4JJEI4
   # Public address:  GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ
-  seed:    SCT4KKJNYLTQO4TVDPVJQZEONTVVW66YLRWAINWI3FZDY7U4JS4JJEI4
+  seed:    SAUHVPR7O7F2QGLDVXG3DQTVHXESE3ZAWHIIGKT35LCHIPLZBZTAFXJA
```

Comment:
Hum the seed doesn't really matter here, but the `Public address:` comment should be in Bech32.

Reply:
Agora crashes/asserts without that change
```
core.exception.AssertError@source/agora/crypto/Key.d(69): SecretKey should always be valid Scalar!
----------------
??:? [0x55a794f0dc85]
??:? [0x55a794f35866]
??:? [0x55a794f18fad]
??:? [0x55a794f0bd69]
/home/dan/dev/bosagora/agora/source/agora/crypto/Key.d:69 [0x55a79488adb7]
/home/dan/dev/bosagora/agora/source/agora/common/Config.d:679 [0x55a7948d5e9e]
/home/dan/dev/bosagora/agora/source/agora/common/Config.d:469 [0x55a7948d4a37]
/home/dan/dev/bosagora/agora/source/agora/common/Config.d:353 [0x55a7948360f5]
/home/dan/dev/bosagora/agora/source/agora/node/main.d:134 [0x55a794835a60]
/home/dan/dev/bosagora/agora/source/agora/node/main.d:131 [0x55a794834760]
??:? [0x55a794f18c7b]
??:? [0x55a794f18b77]
??:? [0x55a794f189cd]
/home/dan/dev/thirdparty/ldc2-1.25.1/bin/../import/core/internal/entrypoint.d:42 [0x55a7948376b4]
??:? __libc_start_main [0x7f65de0ff0b2]
??:? [0x55a79479b59d]


```

Reply:
Please update the comment above. And we probably need to check for valid values on parsing, the assert shouldn't be triggered.